hack/build-go.sh

```diff
@@ -17,7 +17,7 @@ cdup="$(git rev-parse --show-cdup)" && test -n "$cdup" && cd "$cdup"
 
 if [ -z ${VERSION_OVERRIDE+a} ]; then
 	echo "Using version from git..."
-	VERSION_OVERRIDE=$(git describe --abbrev=8 --dirty --always)
+	VERSION_OVERRIDE=$(git rev-parse --verify 'HEAD^{commit}')
```

Comment:
Wouldn't `--verify HEAD` work too?  I had to go look up that syntax.

Are there no other repos which do something similar?  We should try to keep this in common across repos.

Reply:
It's fine by me as is too.